src/Svg.ts

```diff
@@ -774,16 +776,16 @@ export class SVGDrawing {
                                 const ratio = this.config.logoHeight  / this.config.logoWidth;
                                 if(ratio>1){
                                     logoHeight = maxWH;
-                                    logoWidth = logoHeight / ratio; 
+                                    logoWidth = logoHeight / ratio;
                                 }else{
                                     logoWidth = maxWH;
                                     logoHeight = logoWidth * ratio;
                                 }
                             }
                         }
                     }
-                    logoWidth = logoWidth  * logoScale + 2*logoMargin;
-                    logoHeight = logoHeight * logoScale + 2*logoMargin;
+                    logoWidth = logoWidth  * logoScale;
```

Comment:
Are we changing logo dimentions?

Reply:
@shashankraj0202 No, this is to make the vector images consistent with what we have in PNG.

Reply:
Got it